docs/user/openstack/vgpu.md

```diff
@@ -0,0 +1,462 @@
+# Installing a cluster on OpenStack with vGPU support
+
+If the underlying OpenStack deployment have proper GPU hardware installed and
+configured there is a way to pass down vGPU to the pods by using gpu-operator.
+
+
+## Pre-requisites
+
+The following steps are required to be checked before starting the deployment of
+OpenShift.
+
+- Appropriate hardware is installed (like [NVIDIA Tesla
+  V100](https://www.nvidia.com/en-gb/data-center/tesla-v100)) on
+  the OpenStack compute node
+- NVIDIA host drivers installed and nouveau driver removed
+- Compute service installed on it and properly configured
+
+
+## Driver installation
+
+All of the examples assume RHEL8.4 and OSP 16.2 are used.
+
+Given, there is NVIDIA vGPU capable card installed on the machine which intended
+to have compute role, which may be confirmed by using a command which should
+display similar output:
+
+```console
+$ lspci -nn | grep -i nvidia
+3b:00.0 3D controller [0302]: NVIDIA Corporation GV100GL [Tesla V100 PCIe 16GB]
+[10de:1db4] (rev a1)
+```
+make sure to remove `nouveau` driver from loading. It might be necessary to add
+it to `/etc/modprobe.d/blacklist.conf` and/or change grub config:
+
+```console
+$ sudo sed -i 's/console=/rd.driver.blacklist=nouveau console=/' /etc/default/grub
+$ sudo grub2-mkconfig -o /boot/grub2/grub.cfg
+```
+
+After that install host vGPU NVIDIA drivers (which are available to download for
+license purchasers on
+[NVIDIA application hub](https://nvid.nvidia.com/dashboard/)):
+
+```console
+$ sudo rpm -iv NVIDIA-vGPU-rhel-8.4-510.73.06.x86_64.rpm
+```
+
+Note, that drivers version may differ. Be careful to get right RHEL version and
+architecture of the drivers to match installed RHEL.
+
+and reboot the machine.
```

Comment:
Something has got deleted by accident?

Reply:
Nope. Should be a capital letter, or perhaps I rephrase this.